app/src/main/java/com/github/se/polyfit/model/meal/Meal.kt

```diff
@@ -155,11 +127,6 @@ data class Meal(
         val occasion = data["occasion"].let { MealOccasion.valueOf(it as String) }
         val mealTemp = data["mealTemp"] as Double
         val name = data["name"] as String
-
-        val nutritionalInformationData =
-            NutritionalInformation.deserialize(
-                data["nutritionalInformation"] as List<Map<String, Any>>)
-
```

Comment:
I actually don't think we will need to purge it from the DB (unless, for saving space). The data will be there, but, it just won't get used as it gets deserialized. I don't think it will cause any logic issues to leave those as are

Reply:
The `NutritionalInformation` for the meal will be filtered out by the deserializer, as you said. But I meant it more for the sake of uniformity on Firebase.